### ISSUE TITLE
Bug: Integration failing to load in HA 2022.3.0

### ISSUE THREAD

**Issue by Eyco1981** [JurajNyiri/HomeAssistant-Tapo-Control#148]
**Describe the bug**
Integration doesn't load anymore after update to HA Core 2022.3.0.

**To Reproduce**
1. Have Tapo Control installed
2. Update HA Core to 2022.3.0
3. Reboot

**Log**
```
Logger: homeassistant.setup
Source: setup.py:155
First occurred: 21:29:31 (2 occurrences)
Last logged: 21:29:53

    Setup failed for tapo_control: Requirements for tapo_control not found: ['onvif-zeep-async==1.0.0'].
```

```
Logger: homeassistant.util.package
Source: util/package.py:99
First occurred: 22:00:23 (6 occurrences)
Last logged: 22:01:23

Unable to install package onvif-zeep-async==1.0.0: ERROR: Cannot install onvif-zeep-async==1.0.0 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

**Camera (please complete the following information):**
 - Device Model: C110
 - FW: 1.1.12

**Using stream component**
unknown

**Camera works through onvif component OR different cameras work in general**
yes

**Camera has all attributes filled out properly in developer tools**
maybe... 
Device unavailable.
Attributes:
restored: true
icon: mdi:cctv
friendly_name: Camera - HD
supported_features: 3

**HASS Environment**
HA OS on Home Assistant Blue

**Comment by sillvva** [comment]
I had the same error. I tried reinstalling from scratch and got to the part where it asks for IP, username, and password, but it gives the Invalid Authentication error every time, even though the same credentials work in VLC.

![image|50%](https://user-images.githubusercontent.com/5138384/156449185-be966019-aefb-47db-be42-c97f652bbeeb.png)

![image|50%](https://user-images.githubusercontent.com/5138384/156449307-536b3f00-bba9-4561-87c2-1116a203fb04.png)

**Comment by Eyco1981** [comment]
You can use the ONVIF integration or configure a generic camera in YAML to get a camera feed.
Toggeling privacy mode has to wait until the depenency issue is resolved.

**Comment by sillvva** [comment]
I'm not trying to get only the feed. I want to get the controls as well.

As I said, I had exactly the same error message you did (with the dependency). After deleting the integration and reinstalling it, I got to this part.

**Comment by kai74** [comment]
only reinstall from hacks, solved the problem for me

**Comment by lesbass** [comment]
Hi, is it correct you went from version 3.4.4 to 1.4.5?

**Comment by JurajNyiri** [comment]
This has been fixed for [7 days now](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/1.4.5). Please refrain from reporting issues if you are using outdated versions.

**Comment by antonio1475** [comment]
Hi @JurajNyiri 
Just 2 small comments:
1. The "update" didn't show up at all (I check daily in Settings, Add-ons, integrations, etc). Wouldn't it be true that since the "update" is a lower number than the previous version (why isn't it 3.4.5 instead of 1.4.5?), HA didn't show the update available?

As others mentioned, I had to delete the integration, delete the addon, install from scratch. Is that the only update path for everyone?

2. When coming to the Issues tab, it shows Open by default, so one has to remove the filter to see these multiple closed ones - I'm just saying more might come :) 

Thank you for your work, it's very appreciated.

**Comment by JurajNyiri** [comment]
> The "update" didn't show up at all (I check daily in Settings, Add-ons, integrations, etc). Wouldn't it be true that since the "update" is a lower number than the previous version (why isn't it 3.4.5 instead of 1.4.5?), HA didn't show the update available?

Whoops! I totally missed that, updated!

> As others mentioned, I had to delete the integration, delete the addon, install from scratch. Is that the only update path for everyone?

Not anymore, update should be showing up now!

**Comment by Syralist** [comment]
Am I missing something? I don't see a 3.4.5 release.

**Comment by JurajNyiri** [comment]
I have released 3.4.5 as 3.4.7, not sure what happened there. All should be working fine.

**Comment by antonio1475** [comment]
@Syralist for me I only saw it after going to HACS > Integrations > clicking on TAPO Control, then it found the update.

Configuration > 3 dots > Check for updates didn't do it 😅
